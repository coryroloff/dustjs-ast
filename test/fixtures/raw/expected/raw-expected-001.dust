{` this {is} a {#raw /} node`}