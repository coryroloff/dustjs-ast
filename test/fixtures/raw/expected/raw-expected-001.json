{
	"type": "Template",
	"body": [
		{
			"type": "Raw",
			"value": " this {is} a {#raw /} node",
			"start": 0,
			"end": 30,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 30
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Raw",
			"value": " this {is} a {#raw /} node",
			"start": 0,
			"end": 30,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 30
				}
			}
		}
	],
	"start": 0,
	"end": 30,
	"loc": {
		"start": {
			"line": 1,
			"column": 0
		},
		"end": {
			"line": 1,
			"column": 30
		}
	}
}